src/pushNotifications/TokenHandling.ts

```diff
@@ -0,0 +1,75 @@
+import { Platform } from 'react-native';
+
+import { secrets } from '../config';
+import * as SecureStore from 'expo-secure-store';
+import * as appJson from '../../app.json';
+
+
+const namespace = appJson.expo.slug as keyof typeof secrets;
+
+export enum PushNotificationStorageKeys {
+    PUSH_TOKEN = 'PUSH_TOKEN',
+    IN_APP_PERMISSION = 'IN_APP_PERMISSION',
+}
+
+export const handleIncomingToken = async (token?: string) => {
+  // eslint-disable-next-line no-console
+  console.log(token); // FIXME remove for production
```

Comment:
i would suggest to remove the console log already before merge

Reply:
removed in e2f9b849f1157cec42932f4fc81cf4ce13f20fc8